proposals/0024/GOVERNANCE.md

```diff
@@ -102,8 +102,41 @@ The following apply to the repo or the part of the codebase for which one would
 - Assigned bugs related to area of expertise
 - May get a badge on PR and issue comments
 
+## Inactive members
+
+>This inactive members section follows the guideline defined in the Kubernetes community<sup>[5]</sup>
+
+_Members are continuously active contributors in the community._
+
+A core principle in maintaining a healthy community is encouraging active
+participation. It is inevitable that people's focuses will change over time and
+they are not expected to be actively contributing forever.
+
+However, being a member of the Tinkerbell GitHub organization comes with
+an [elevated set of permissions]. These capabilities should not be used by those
+that are not familiar with the current state of the Tinkerbell project.
+
+Therefore members with an extended period away from the project with no activity
+will be removed from the Tinkerbell Github Organization and will be required to
+go through the org membership process again after re-familiarizing themselves
+with the current state.
+
+### How inactivity is measured
+
+Inactive members are defined as members of the Tinkerbell Organization
+with **no** contributions across any repos within 8 months. This is
+measured by the CNCF [DevStats project].
```

Comment:
Do we have any mechanism for this? Do we have a job we can run periodically to diff a list of usernames vs recent contributions? Maybe we don't auto-remove, but even if its a monthly job that just scans for activity and reports inactive contributors

Reply:
Opened tinkerbell/org#7